client/src/components/layers/EditAnnotationLayer.ts

```diff
@@ -126,6 +126,11 @@ export default class EditAnnotationLayer extends BaseLayer<GeoJSON.Feature> {
     if (this.featureLayer) {
       this.featureLayer.removeAllAnnotations();
       this.featureLayer.mode(null);
+      if (this.selectedHandleIndex !== -1) {
+        this.selectedHandleIndex = -1;
+        this.hoverHandleIndex = -1;
+        this.$emit('update:selectedIndex', this.selectedHandleIndex);
+      }
```

Comment:
Do you need to check that this is a non-rect before doing the emit like on line 75?

Reply:
It's specifically setting it to -1, so even if it is a `rect` that is okay (and most likely preferred to make sure we are at the default).  Line 75 was emitting actual selected handles, not a -1.

Reply:
Oh duh.  Sorry